examples/step-33/step-33.cc

```diff
@@ -2368,7 +2368,6 @@ namespace Step33
       grid_in.read_ucd(input_file);
     }
 
-    dof_handler.clear();
```

Comment:
I considered this line as a typo since it is just the first usage of `dof_handler` in the code. The flow is quite transparent here: `main()` --> `run()` --> `dof_handler.clear()`.